README.md

```diff
@@ -42,11 +42,9 @@ following attributes need to be specified:
 -   Server URL: CloudBees CD Server URL
 
 -   REST API Version: CloudBees CD Server Rest API Version
-
--   User Name: CloudBees CD user name for Connection
-
--   User Password: CloudBees CD password for Connection
-
+-   Credentials Type:
+    -   Username and Password: CloudBees CD username and passport for Connection
+    -   Stored Credential: Used to configured authentication via stored credentials, which can be username/password or secret types, such as tokens 
```

Comment:
configure instead of configured
 secret text, not types - exact supported type of stored credential
 -   Stored Credential: Used to configure authentication via stored credentials, which can be username/password or secret text, which contains SSO session ID token